### PR TITLE
Tweak unique image tags to contain date

### PR DESCRIPTION
**Pull request by c-nixon**
This changes the Makefile to include "today"'s date in the unique docker image tag

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -222,7 +222,7 @@ build-image: ## Build a docker image as specified in the Dockerfile
 
 .PHONY:publish-image
 publish-image: ## Publish SemVer compliant releases to our registroies
-	$(eval TARGET_VERSIONS := $(BUILD_VERSION) $(shell if [ "$(BETA_VERSION)" = "0" ]; then echo "$(BUILD_VERSION)-build.$(shell docker images -q $(REPO):$(BUILD_TAG)) $(MAJOR_VERSION) $(MAJOR_VERSION).$(MINOR_VERSION) latest"; fi))
+	$(eval TARGET_VERSIONS := $(BUILD_VERSION) $(shell if [ "$(BETA_VERSION)" = "0" ]; then echo "$(BUILD_VERSION)-$(shell date '+%Y-%m-%d').$(shell docker images -q $(REPO):$(BUILD_TAG)) $(MAJOR_VERSION) $(MAJOR_VERSION).$(MINOR_VERSION) latest"; fi))
```
Comment (jmoses) [inline Makefile:225]: Is this better as a variable with the rest of the BUILD_* ones (https://github.com/logdna/logdna-agent-v2/pull/44/files) and then included the same way?

**Review by jmoses — APPROVED** [review]
One question, if the answer is "no", that's fine.

File: Makefile (hunk 1 above)
Comment (c-nixon) [reply Makefile:225]: As in extracting the date out as a separate variable?

Comment (jmoses) [reply Makefile:225]: Yeah, like, there's a BUILD_TIME one, or something.

Comment (jmoses) [reply Makefile:225]: Oh that link totally doesn't go where I meant. https://github.com/logdna/logdna-agent-v2/blob/master/Makefile#L35

**Review by jakedipity — APPROVED** [review]

**Comment by jakedipity** [comment]
@logdnabot test this

**Review by jmoses — APPROVED** [review]

**Pull request merged**